pdr_backend/models/prediction.py

```diff
@@ -11,6 +11,8 @@ def __init__(
         source,
         payout,
         user,
+        submittimestamp,
+        predictionId,
```

Comment:
Any reason this is not `prediction_id` ? (Snake case)

Reply:
solved